.github/PULL_REQUEST_TEMPLATE/python.md

```diff
@@ -1,34 +1,46 @@
-**Changes:**
+Changes
+-------
 
 * IoT3 SDK:
-    * core: ...
-    * mobility: ...
+    * core: new awesome method
+    * mobility: fixed something that was broken
 * applications:
-    * iqm: ...
-    * ...
+    * info: new stuff
+    * interqueuemanager: update some message version
+    * quadkeys: nothing
+    * status: add a new connection life status
+    * vehicle: change the frequency
 
-Closes: #
+Close #XXX
 
+Close #YYY
+
+Test
 ---
-**How to test:**
 
-_**Note:** in the following, lines starting with `$` are to be executed on your machine, as a non-root user; lines starting with `(docker)$` are to be executed in the Docker container; lines starting with `(docker)🐍 $` are to be executed in the Docker container, in the python venv._
+## How to test
```

Comment:
The rendering for the `Test` and `## How to test` is a bit odd.

Ideally, we'd use the same markup for all titles, and not mix the prefix `##` with the underlines, as @nbuffon pointed out.